traveler_app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
           package="com.guestlogix.traveler">
 
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Should this be in the SDK manifest? What happens if we dont have these permissions?

Reply:
✓